src/helpers/mapHelper.js

```diff
@@ -0,0 +1,19 @@
+import { Platform } from 'react-native';
+
+export function locationString(address) {
+  if (!address) return '';
+
+  return encodeURIComponent(address);
+}
+
+// a maps link is different between the platforms
+export function locationLink(mapsString) {
+  switch (Platform.OS) {
```

Comment:
we have a Platform check in configs device, we use everywhere. we can also use that here.